bliss/encoder/unconstrained_dists.py

```diff
@@ -35,6 +35,19 @@ def get_dist(self, params):
         return Normal(mean, sd)
 
 
+class ShearUnconstrainedNormal:
```

Comment:
It be better if this distribution had a name that wasn't specific to shear, so that it can be reused for other applications